### PR TITLE
Azfw policy creation

### PR DESCRIPTION
**Pull request by amahajan-75**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-azurefirewall-create-with-firewallpolicy-apprule-netrule-ipgroups/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This template deploys an Azure Firewall with Firewall Policy (including multiple application and network rules) referencing IP Groups in application and network rules.
```
Comment (bmoore-msft) [inline 101-azurefirewall-create-with-firewallpolicy-apprule-netrule-ipgroups/README.md:1]: See contribution guide for a proper README.md example

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Also check the pipeline results for other changes that need to be made

**Comment by bmoore-msft** [comment]
readme looks good - take a look that the pipeline results for other changes that need to be made...

File: 101-azurefirewall-create-with-firewallpolicy-apprule-netrule-ipgroups/README.md (hunk 1 above)
Comment (amahajan-75) [reply 101-azurefirewall-create-with-firewallpolicy-apprule-netrule-ipgroups/README.md]: Changes has been made to README file as per guidance.

**Review by amahajan-75 — COMMENTED** [review]
Changes has been made to the respective templates as per pipeline results

**Pull request merged**